src/users/users.controller.ts

```diff
@@ -100,6 +94,7 @@ export class UsersController {
 
   @ApiBody({ type: CredentialsDto })
   @AllowAny()
+  @UseInterceptors(UserSettingsInterceptor)
```

Comment:
I do not think we need this interceptor on login endpoint.

Reply:
It's removed in the following commit